docs/UI/PerformInteraction/index.md

```diff
@@ -39,6 +39,7 @@ A UI.PerformInteraction with a timeout value of `0` should not be timed out imme
 |timeout|Integer|true|minvalue: 5000<br>maxvalue: 100000|
 |interactionLayout|[Common.LayoutMode](../../common/enums/#layoutmode)|false||
 |appID|Integer|true||
+|cancelID|Integer|false|can be used later with CancelInteraction RPC to specifically close this interaction|
```

Comment:
This table does not have a description section

Reply:
fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1f93da0ff41d60f00c4d1bb3cc93ffeb5bd034e0